rust-toolchain

```diff
@@ -1 +1 @@
-1.57.0
+1.63.0
```

Comment:
This should ideally be the MSRV so we don't accidentally get any incompatibility issues
```suggestion
1.57.0
```